src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -1,10 +1,80 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.util.FiringSolution;
+
+import static frc.robot.Constants.Shooter.*;
+import static frc.robot.UnitConversions.Shooter.*;
 
+/**
+ * Represents the robot's shooter
+ * 
+ * <p>
+ * The mechanism contains a single flywheel with a fixed hood and a
+ * pneumatically actuated "deflector" that redirects the shot to a
+ * more horizontal trajectory
+ */
 public class ShooterSubsystem extends SubsystemBase {
+
+    // Flywheel motors
+    private WPI_TalonFX flywheelLeader;
+    private WPI_TalonFX flywheelFollower;
+
+    private DoubleSolenoid deflectorSolenoid;
+
     public ShooterSubsystem() {
+        // Configure the flywheel motors
+        flywheelLeader = new WPI_TalonFX(LEADER_MOTOR_ID);
+        flywheelLeader.configFactoryDefault();
+        flywheelLeader.setNeutralMode(NeutralMode.Coast);
+        FLYWHEEL_MOTOR_PID.configureMotorPID(flywheelLeader);
+
+        flywheelFollower = new WPI_TalonFX(FOLLOWER_MOTOR_ID);
```

Comment:
If you find yourself adding/removing/changing configurations to the motors, break these out into a sub function where you pass in the ID and PID, so you only have to change the similar code in one place, and then can call the function twice. This just leaves less room for error, and saves time.

If this isn't going to change, it's fine to leave.